temporal_sqlalchemy/metadata.py

```diff
@@ -1,29 +1,14 @@
 import sqlalchemy.orm as orm
 
-TEMPORAL_METADATA_KEY = '__temporal'
-
 __all__ = [
-    'get_session_metadata',
-    'set_session_metadata',
+    'STRICT_MODE_KEY',
+    'CHANGESET_STACK_KEY',
+    'IS_COMMITTING_KEY',
+    'IS_VCLOCK_UNCHANGED_KEY',
 ]
 
 
-def set_session_metadata(session: orm.Session, metadata: dict):
-    if isinstance(session, orm.Session):
```

Comment:
I removed these functions since everything is being stored at the top `session.info` dict.